src/components/Delete/Delete.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Modal, ModalActions, ModalContent } from '@dhis2/ui'
+import ButtonItem from '../ButtonItem/ButtonItem'
+import i18n from '../../locales/index.js'
+import { deleteCatchmentJob } from '../../api/crosscutRequests'
+
+function Delete(props) {
+    const { setShowDelete, toggle, id } = props
+
+
+    const handleDelete = async () => {
+        await deleteCatchmentJob(id)
+        toggle()
```

Comment:
Just curious: what does `toggle()` do?

Reply:
toggle triggers the fetch to get catchments